### PR TITLE
Make task

### PR DESCRIPTION
**Pull request by alex39292**

### CODE REVIEW

File: Data Types/explicit.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+let a = 1;
```
Comment (SiarheiRylach) [inline Data Types/explicit.js]: use `const` if you won't use for variable `=` operator furher

File: Data Types/multiplication.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+let a = 123;
+let b = 'some text';
+let c = true;
+
+//boolean * string
+console.log(c * b); //truesome text
```
Comment (SiarheiRylach) [inline Data Types/multiplication.js]: NaN

File: Data Types/multiplication.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+//number * boolean
+console.log(a * c); //124
```
Comment (SiarheiRylach) [inline Data Types/multiplication.js]: 123

File: Data Types/multiplication.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+//number * string
+console.log(a * b); //123some text
```
Comment (SiarheiRylach) [inline Data Types/multiplication.js]: NaN

File: Data Types/multiplication.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+//string * number
+console.log(b * a); //some text123
```
Comment (SiarheiRylach) [inline Data Types/multiplication.js]: NaN

File: Data Types/multiplication.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+//string * string 
+console.log(b * b); //some textsome text
```
Comment (SiarheiRylach) [inline Data Types/multiplication.js]: NaN

File: Data Types/explicit.js (hunk 1 above)
Comment (alex39292) [reply Data Types/explicit.js]: fix, use const in all files where needed

File: Data Types/multiplication.js (hunk 2 above)
Comment (alex39292) [reply Data Types/multiplication.js]: Fix comments

File: Data Types/multiplication.js (hunk 3 above)
Comment (alex39292) [reply Data Types/multiplication.js]: fix comment

File: Data Types/multiplication.js (hunk 4 above)
Comment (alex39292) [reply Data Types/multiplication.js]: fix comment

File: Data Types/multiplication.js (hunk 5 above)
Comment (alex39292) [reply Data Types/multiplication.js]: fix comment

File: Data Types/multiplication.js (hunk 6 above)
Comment (alex39292) [reply Data Types/multiplication.js]: fix comment

**Pull request merged**